meson.build

```diff
@@ -17,9 +17,12 @@ conf_data.set('libmt32emu_EXPORTS_TYPE',  mt32emu_exports_type)
 subdir('mt32emu/src')     # all source files
 subdir('include/mt32emu') # public interface
 
+mt32emu_lib_includes = include_directories('include/mt32emu')
+mt32emu_dep_includes = include_directories('include')
+
 mt32emu_lib = library('mt32emu', mt32emu_sources,
-                      include_directories : include_directories('include/mt32emu'),
+                      include_directories : mt32emu_lib_includes,
                       cpp_args : ['-DMT32EMU_WITH_INTERNAL_RESAMPLER'])
 
-mt32emu_dep = declare_dependency(include_directories : include_directories('include'),
+mt32emu_dep = declare_dependency(include_directories: mt32emu_dep_includes,
```

Comment:
Good change :)

Just please put a space between `include_directories` and `:` in here (to keep the style consistent with Meson default coding style).